.github/workflows/build.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on: 
+  push:
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Python ${{ matrix.version }}
+    strategy: 
+      matrix:
+        version: [2.7, 3.4, 3.5, 3.6]
+
+    steps: 
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.version }}
+      - name: Install Dependencies
+        run: | 
+          python -m pip install --upgrade pip
+          pip install flake8 pytest
+          pip install mock
```

Comment:
Any idea why the travis config didn't need these additional steps?

Reply:
Really not sure! :/